classes/Tools.php

```diff
@@ -408,20 +408,20 @@ public static function secureReferrer($referrer)
     * Get a value from $_POST / $_GET
     * if unavailable, take a default value
     *
-    * @param string $key Value key
-    * @param mixed $default_value (optional)
-    * @return mixed Value
+    * @param string     $key          Value key
+    * @param mixed|null $defaultValue (optional)
+    * @return mixed|null Value
     */
-    public static function getValue($key, $default_value = false)
+    public static function getValue($key, $defaultValue = null)
     {
         if (!isset($key) || empty($key) || !is_string($key)) {
-            return false;
+            return $defaultValue;
```

Comment:
Hello @gskema 
I'm closing this PR because this behavior can break a lot of thing.. Change the return here is dangerous (it makes sense but..)

Regards